Expansion Map Elements

DerSchlupf opened the pull request:
**WiP!!**

To do for the three documents in this PR:

### **map-elements:**
TO-DO:
- [x] asset elemenatl map tiles
- [x] assets location token
- [ ] proofreading (also Links, Index, etc.)
- [ ] missing clarifactions: hero position on elemantel map / discvover between surface & subterranean

<img width="1241" height="1754" alt="en-29" src="https://github.com/user-attachments/assets/fb8d6d9b-8807-4320-bea4-693853e2f728" />
<img width="1241" height="1754" alt="en-30" src="https://github.com/user-attachments/assets/73b3b324-16c1-471f-9c62-3be943953c6a" />

### **expansion content:**
TO-DO:
- [x] decide which assets and include them (possible ones: creature bank token/backside; stack token; creature banks unit cards; creature bank combat setup)
- [x] general layout
- [ ] proofreading (also Links, Index, etc.)

<img width="1241" height="1754" alt="en-49" src="https://github.com/user-attachments/assets/30f4f7cf-ad4d-4ed7-bc09-2190906ce65e" />
<img width="1241" height="1754" alt="en-50" src="https://github.com/user-attachments/assets/09d272e5-d7f8-435a-bb6d-25077750356f" />

### **all map locations:**
TO-DO:
- [x] fill space next to tavern
- [x] repair storm elemantal art (p.56) and cyclops (p 59.)
- [x] include padoras box variant
- [x] make location pics for whirlpool and creature banks (p.60)
- [x] add the right texts to whirlpool and creature bank introduction
- [ ] proofreading (also Links, Index, etc.)
- [ ] clarifications: whirlpool "losing unit", monoloith MP cost; pandoras box card;

<img width="1241" height="1754" alt="en-61" src="https://github.com/user-attachments/assets/9999898c-6025-4521-ae58-bf919e9c385e" />
<img width="1241" height="1754" alt="en-62" src="https://github.com/user-attachments/assets/7c2e7ff4-3881-4406-bbe7-c6c29fdd633c" />
<img width="1241" height="1754" alt="en-63" src="https://github.com/user-attachments/assets/6a6bc3ea-86e0-41f1-83c2-05b0f2d96aa6" />

DerSchlupf commented:
By the way i think these assets should we add: 
<img width="343" height="204" alt="image" src="https://github.com/user-attachments/assets/975e6e42-be2e-406a-935e-83bfe9629ba7" />
<img width="344" height="275" alt="image" src="https://github.com/user-attachments/assets/68cfdc1d-6770-4518-857d-9ac3fca6d197" />

qwrtln commented:
Added assets, how's that?

<img width="1241" height="1754" alt="en-04" src="https://github.com/user-attachments/assets/df5736e1-d917-41ad-9056-c1c753a8e1c9" />
<img width="1241" height="1754" alt="en-05" src="https://github.com/user-attachments/assets/bbb70f4d-6ec8-4050-a4e1-e3c213229b2f" />

DerSchlupf commented:
Some ideas:

1) replace "stronghold map tile" with "surface map tile" cause the difference between surface and subterranean is relevant, not the expansion, where the tile comes from. 

2) should we list the token heading at first? It's kind of a general rule and after that we write something about specific tokens (whirlpools, gate...).

3) We could add a sentence about tokens under the "discover and placing tiles" heading like "Apply further rules, when discover a map tile with location tokens". (and add a link to the token chapter).  But probably it's not necessary.

4) I will push a suggestion about including creature banks and elemental map tiles here as well soon.

DerSchlupf commented:
--> changed the order, added heading location token, added whirpoll (without asset unfortunatly) and create pagetarget at creature bank table (red marking) 
--> removed firebird art, but there is space for new art next to tavern.

Should i push it? 

<img width="1295" height="896" alt="image" src="https://github.com/user-attachments/assets/dadefaa3-faf1-4e04-89fd-7b6259471440" />
<img width="1288" height="891" alt="image" src="https://github.com/user-attachments/assets/8939ac12-c5f4-45fb-9c9f-54c10dd8a371" />
<img width="1292" height="905" alt="image" src="https://github.com/user-attachments/assets/6f88005a-0f00-4e00-9dda-e0e786410d31" />
<img width="1293" height="902" alt="image" src="https://github.com/user-attachments/assets/01e17934-7174-4021-be61-66c82d5cd9c6" />

DerSchlupf commented:
Also in map elements: 
- changed the order (the general Token Text i would put first, wdyt?)
- renamed stronghold map til in "Surface Map Tile"
- whirlpool text removed (is now in all map location) 
- made subterra gate text shorter, but not removed cause the rules are more tricky.
- added couple of pagetargets and -links between all map location and map elements (and creature bank expansion content)

Push?
<img width="1284" height="888" alt="image" src="https://github.com/user-attachments/assets/78c79ecb-9370-4689-900a-9f622bf7a919" />

DerSchlupf commented:
There is supposed to be one "}" too many in line 751 or what don't i understand. I found no error in the code.  Probably you can repair it.

DerSchlupf commented:
By the way: The way it's structured now, it might make more sense to swap the two sides, so map tiles first and tokens afterward. Like you've suggested before :D Decide you what you like more.

DarthGandalf commented:
The order of images of elemental tiles doesn't match the order of their labels. I think we should update the asset to swap the air and water tiles. It's possible to swap labels instead, but that removes consistency with other labeled images throughout the book, which are in 2 columns

Thoughts?

DarthGandalf commented:
Or even like this:
<img width="1240" height="1754" alt="blah" src="https://github.com/user-attachments/assets/43681b15-d61f-46d2-987b-2caef82b713a" />

DerSchlupf commented:
Suggestion for text changes:

**Discover token:** 

Location token are added by several expansions to the game.  They are usually placed on specific fields of the map, when a hero discovers the associated map tile and overwrite the original location they are placed on. But they also might have an own position in the map layout, depending on the chosen scenario. Location Tokens *cannot* be placed on blocked fields, other location tokens or fields containing locations required to meet any of the scenarios victory condition. Keep this rules in mind when rotating map tiles with location tokens.

**Subterranean gate:**

Subterranean gate token are also determined by the scenarios map layout and they connect both a surface map tile and an adjacent subterranean map tile. After discovering the first of the corresponding map tiles you may choose the position of the subterranean gate token as long as it connects the two indicated map tiles and without covering forbidden fields. The scenario map Layout only shows  **which two tiles are connected**, not the exact fields that must be connected.

**Creature banks**

As an **optional rule** you can decide to play with creature banks. If you do each time a player discovers a far or near map tile they may decide to draw a creature bank token corresponding to the discovered map tile and – contrary to the normal rules – place it on its blocked field.

DerSchlupf commented:
And regarding subterrain gate in all map location:
![Screenshot_20250804_155053_com_android_chrome_ChromeTabbedActivity](https://github.com/user-attachments/assets/c90a2b4b-91c2-483d-9343-9769516ade41)

That part with "move to the connected location..." makes less sense. It refers to archons wording with "entrance" and "exit" of such a gate what we skipped in the rewrite. 

**Possible other wording:**

Connects a surface and a subterranean Map Tile (Link to subterranean map tiles). Treat both fields of the gate as single field. If the connected Tile isn't discovered, discover it. Otherwise treat this location as empty field.

DerSchlupf commented:
And last but not least regarding this:
![Screenshot_20250804_222303_com_android_chrome_ChromeTabbedActivity~2](https://github.com/user-attachments/assets/0bea6388-8bc3-48fb-b58a-6acb6bfbb851)

I would suggest to add "drawing a token", e. g.:

... you may choose to draw a Creature Bank Token of the corresponding type and place it on one of that's tiles blocked fields.

(or something similar)
